### PR TITLE
Minor logging changes from Puzzle Boat 10 (2023)

### PR DESCRIPTION
**Pull request by akirabaruah**

### CODE REVIEW

File: cardboard/settings.py [hunk 1]
```diff
@@ -311,6 +311,10 @@ class TaskPriority(Enum):
         },
     },
     "loggers": {
+        "": {
+            "handlers": ["console"],
+            "level": os.getenv("DJANGO_LOG_LEVEL", "INFO"),
```
Comment (rgossiaux) [inline cardboard/settings.py:316]: I think this env variable is supposed to be specific to the logging messages that django itself generates so I don't think we should reuse it here as this is for the logging messages we created ourselves. IMO better to either hardcode it, condition it on the DEBUG setting, or create a separate env variable

File: cardboard/settings.py [hunk 2]
```diff
@@ -311,6 +311,10 @@ class TaskPriority(Enum):
         },
     },
     "loggers": {
+        "": {
```
Comment (rgossiaux) [inline cardboard/settings.py:314]: it looks like you can alternatively pull this out to the top level in a `root` entry a la https://docs.djangoproject.com/en/5.0/topics/logging/#id3 

don't think there's any functional difference between the two

Comment (akirabaruah) [reply cardboard/settings.py:314]: Yep. Done.

File: cardboard/settings.py (hunk 1 above)
Comment (akirabaruah) [reply cardboard/settings.py:316]: Sounds good. I just changed the variable to `DJANGO_APP_LOG_LEVEL` for flexibility in the future.

**Review by rgossiaux — APPROVED** [review]
sweet, thanks!

**Pull request merged**